### ISSUE TITLE
No docker image for 3.1.1

### ISSUE THREAD

**Issue by jcstanaway** [powerfulseal/powerfulseal#312]
Various places (docs, deployment manifest) refer to image `powerfulseal/powerfulseal:3.1.1`.  However, this image does not appear to exist.  There are no images tagged at https://hub.docker.com/_/powerfulseal.  The most recent image that appears to be available is `bloomberg/powerfulseal:3.1.0` (https://hub.docker.com/r/bloomberg/powerfulseal/tags). The docs refer to https://hub.docker.com/r/powerfulseal/powerfulseal/tags, but that's 404.

**Comment by seeker89** [comment]
Yes, sorry about that. PowefulSeal is in a process of moving into its own organization, and we've had some trouble setting up the CI pipeline. This shouldn't take much longer now.

**Comment by dza89** [comment]
@seeker89 It's still not there..

**Comment by seeker89** [comment]
Yes. Apologies for the delay. We're blocked by some paperwork...

**Comment by sathiyajith** [comment]
When will the image be created? Is there a slack community for this tool? If so, can you add us and give update about this there?

**Comment by QAInsights** [comment]
Any updates on this please? Thanks!
